src/langsmith/agent-builder-slack-install.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Install LangSmith Agent Builder for Slack
+description: Connect the LangSmith Agent Builder to your Slack workspace to power AI agents.
+sidebarTitle: Slack installation
+mode: wide
+---
+
+The LangSmith Agent Builder Slack app enables your agents to interact with your Slack workspace.
+
+## How to install
+
+To install the LangSmith Agent Builder for Slack:
+
+1. Navigate to Agent Builder in your [LangSmith workspace](https://smith.langchain.com)
+2. Create or edit an agent
+3. Add Slack as a trigger or enable Slack tools
+4. When prompted, authorize the Slack connection
+5. Follow the OAuth flow to grant permissions to your Slack workspace
+
+The app will be installed automatically when you complete the authorization.
+
+## Permissions
+
+The LangSmith Agent Builder requires the following permissions:
+
+- **Send messages** - Send direct messages and post to channels
+- **Read messages** - Read channel history and thread messages
+- **View channels** - Access basic channel information
+- **View users** - Look up user information for messaging
+
+These permissions enable agents to communicate effectively within your workspace.
```

Comment:
```suggestion
These permissions enable agents to communicate effectively within your Slack workspace.
```
Assume Slack workspace here. Should probably keep which workspace we're talking about clear.